src/WorkerMessageTypes.d.ts

```diff
@@ -72,6 +72,22 @@ export interface DecryptLocalKeysWorkerResponse {
     };
 }
 
+export interface RotateUserPrivateKeyWorkRequest {
+    type: "ROTATE_USER_PRIVATE_KEY";
+    message: {
+        passcode: string;
+        privateKey: Uint8Array;
```

Comment:
This should probably be named `encryptedPrivateUserKey` to be more clear about what it represents.